rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -1589,6 +1589,103 @@
         "minLength": 4
```

Comment:
This files get bigger and bigger, maybe delete it and put it in .gitignore. The setup script should recreate it if the date is older as source file.

Reply:
ok